src/VariantSet.js

```diff
@@ -308,17 +304,16 @@ export class SearchVariants extends Component {
     return (
       <span>
         <div>
-          <span className="searchBar">reference name: <input id="refName" size="5" defaultValue="1" type="text"/></span>
+          <span className="searchBar">reference name: </span>
```

Comment:
```
      <TextInput onchange={function() { this.setState({end: this.value})}} title="Reference name" />

```
      
      new component type textentry --
      you can pass a callback for onchange to it